### ISSUE TITLE
Not working third dot

### ISSUE THREAD

**Issue by davydkoff** [smashingboxes/OwlCarousel2#1191]
http://www.owlcarousel.owlgraphic.com/demos/responsive.html

**Comment by massic80** [comment]
Position must be checked AFTER the transition, not before

**Comment by davydkoff** [comment]
No, I mean, the third point is not assigned to a class "active", when checked

**Comment by massic80** [comment]
Mine was a suggestion for developers ;)
I have no time for pushing the modification, but it's the cause of the trouble
